installation/linux.md

```diff
@@ -48,17 +48,17 @@ Alternatively resort to the [manual installation approach](linux.html#manual-ins
 Choose between the latest Beta release or a Snapshot with all incoming contributions, rebuild on [every change](https://oss.jfrog.org/webapp/#/builds/openHAB-Distribution).
 As openHAB 2 is still in an evolving state, the snapshot may be the **preferred choice**.
 
-The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab.list`.
+The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab2.list`.
 Be careful to not have conflicting repositories in your sources list.
```

Comment:
Do you think there should be a mentioning that the user may need to delete `openhab.list`? Yes, that should be part of the migration tutorial but I'm almost certain this will hit users migratin usin their "best knowledge" and this guide.

If yes, needs to be changed further down too